lib/school_house/content/post.ex

```diff
@@ -16,7 +16,11 @@ defmodule SchoolHouse.Content.Post do
   ]
 
   def build(filename, attrs, body) do
-    slug = Path.basename(filename, ".md")
+    slug =
+      filename
+      |> Path.basename(".md")
+      |> String.slice(11..-1)
```

Comment:
Within the function it may be nice to assign 11 to a variable and add a comment explaining it, something like:

```elixir
# trim the date prefix to get the slug for the blog post
# i.e. 2021_06_15_really_smart_blog_post -> really_smart_blog_post
date_prefix_length = 11
```

Reply:
This is a great suggestion! I'll make this change later 🎉